src/client/common/application/notebook.ts

```diff
@@ -114,10 +114,7 @@ export class VSCodeNotebook implements IVSCodeNotebook {
         serializer: NotebookSerializer,
         options?: NotebookDocumentContentOptions
     ): Disposable {
-        return workspace.registerNotebookSerializer(notebookType, serializer, options, {
-            displayName: 'Jupyter',
-            filenamePattern: ['*.ipynb']
-        });
+        return workspace.registerNotebookSerializer(notebookType, serializer, options);
```

Comment:
That's odd. Had to spelunk around in vscode core to see why this was an issue. Wonder why that option is even here? https://github.com/microsoft/vscode/blob/7adcbfdd68925543df362b74c7026017d0af3b24/src/vs/workbench/api/browser/mainThreadNotebook.ts#L119